### DISCUSSION TITLE
Weighting / priorization for multi objectives

### DISCUSSION THREAD

**Discussion by DeLaRiva** [optuna/optuna#3845] [Ideas]
It would be helpful if one could weight or prioritize between the to be optimized variables. For example, the costs for building goods have to be optimized in respect to the quality of them but, in doubt, the costs are more important.

Tbh, I don't know if this is feasible within this (pareto-)framework but I wanted to drop my thoughts just quickly.

**Comment by nzw0301** [comment]
Do you know some concrete references, ideally research papers for such a technique?

**Comment by DeLaRiva** [comment]
No, sorry, I'm not really from this domain but grassing there for once and a while :)

In some other methods or algorithms I simply used multiplying the variables with 0.x factors like 0.7 * building_costs + 0.3 * building_quality.
Thinking about a Pareto front one could maybe shift the best optimum from the Pareto front to the left or right on one axis?
